test/e2e/cert_policy_test.go

```diff
@@ -179,6 +178,26 @@ var _ = Describe("Test cert policy", func() {
 			common.DoRootComplianceTest(certPolicyName, policiesv1.Compliant)
 		})
 		AfterAll(func() {
+			By("the policy should have matched history after all these test")
+			common.DoHistoryUpdatedTest(certPolicyName,
+				"Compliant",
+				"NonCompliant;  1 certificates defined SAN entries do not match pattern Allowed: "+
+					".*.test.com Disallowed: [\\*]: default:cert-policy-secret",
+				"Compliant",
+				"NonCompliant;  1 certificates defined SAN entries do not match pattern Allowed: "+
+					".*.test.com Disallowed: [\\*]: default:cert-policy-secret",
+				"Compliant",
+				"NonCompliant;  1 CA certificates exceed the maximum duration of 26280h0m0s: "+
+					"default:cert-policy-secret",
+				"Compliant",
+				"NonCompliant;  1 certificates exceed the maximum duration of 9528h0m0s: "+
+					"default:cert-policy-secret",
+				"Compliant",
+				"NonCompliant;  1 CA certificates expire in less than 45h0m0s: default:cert-policy-secret",
+			)
```

Comment:
Since this is a test on its own, I think it should be in an `It`, not inside the `AfterAll`. Since this `Describe` block has the `Ordered` attribute, the `It` statements will always be done in the right order. `AfterAll` should just be for cleanup that always needs to take place, even when a test fails.

Reply:
ok!